### PR TITLE
Replaces shipside chemical machines with a vendor

### PR DESCRIPTION
**Pull request by Zonespace27**
# About the pull request

Removes the all chemistry machines from all shipside maps. Adds a vendor that can vend the following mixes as pill bottles:
- IA
- MB
- KD
- Iron
- NW
- D+

The vendor can be used thrice.

Additionally, adds tricord bottles to the wey-meds.

# Explain why it's good for the game

I believe chemistry is a fundamentally flawed system for PvE, for the following reasons:

- The time corpsmen spend in the chemlab detracts from potential roleplay they could have. From my time GMing, I can attest to them regularly sitting in the chemlab up to and sometimes past briefing start.
- The infinite amount of quite-good chemicals that corpsmen can produce makes medical attrition harder as a GM.
- The gameplay of chemistry isn't at all fun, it's just rote instruction-following. 

Replacing it with a limited-use vendor is good for the following reasons:
- It reduces the amount of time that corpsmen are holed up in their medbay.
- It forces corpsmen to have to make decisions on what specialized chemicals to bring on a mission, encouraging thoughtful planning.

The tricord change is because it's a fairly cheap, common (and not great) chemical that's no longer accessible, so I thought it would be best in the wey-med.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

image

</details>

# Changelog

:cl:
add: Added a points-based vendor to all shipmaps for obtaining normally chemistry-only pill bottles.
add: Added tricord pill bottles to weymeds
del: Removed all chemical machines from all shipmaps.
/:cl:

### CODE REVIEW

File: code/game/machinery/vending/vendor_types/squad_prep/squad_medic.dm [hunk 1]
```diff
@@ -336,3 +336,33 @@ GLOBAL_LIST_INIT(cm_vending_clothing_forecon_medic, list(
...
+// Chemical vendor
+
+GLOBAL_LIST_INIT(cm_vending_chemical_medic, list(
+		list("PILL BOTTLES", 0, null, null, null),
+		list("Pill Bottle (Imidazoline-Alkysine)", 60, /obj/item/storage/pill_bottle/imialk, null, VENDOR_ITEM_REGULAR),
+		list("Pill Bottle (Meralyne-Bicardine)", 60, /obj/item/storage/pill_bottle/merabica, null, VENDOR_ITEM_REGULAR),
+		list("Pill Bottle (Kelotane-Dermaline)", 60, /obj/item/storage/pill_bottle/keloderm, null, VENDOR_ITEM_REGULAR),
+		list("Pill Bottle (Nitrogen-Water)", 60, /obj/item/storage/pill_bottle/nitrogenwater, null, VENDOR_ITEM_REGULAR),
+		list("Pill Bottle (Dexalin+)", 60, /obj/item/storage/pill_bottle/dexalinplus, null, VENDOR_ITEM_REGULAR),
+		list("Pill Bottle (Iron)", 60, /obj/item/storage/pill_bottle/iron, null, VENDOR_ITEM_REGULAR),
```
Comment (Zonespace27) [inline code/game/machinery/vending/vendor_types/squad_prep/squad_medic.dm:350]:
```suggestion
		list("PILL BOTTLES", 0, null, null, null),
		list("Pill Bottle (Imidazoline-Alkysine)", 40, /obj/item/storage/pill_bottle/imialk, null, VENDOR_ITEM_REGULAR),
		list("Pill Bottle (Meralyne-Bicardine)", 40, /obj/item/storage/pill_bottle/merabica, null, VENDOR_ITEM_REGULAR),
		list("Pill Bottle (Kelotane-Dermaline)", 40, /obj/item/storage/pill_bottle/keloderm, null, VENDOR_ITEM_REGULAR),
		list("Pill Bottle (Nitrogen-Water)", 40, /obj/item/storage/pill_bottle/nitrogenwater, null, VENDOR_ITEM_REGULAR),
		list("Pill Bottle (Dexalin+)", 40, /obj/item/storage/pill_bottle/dexalinplus, null, VENDOR_ITEM_REGULAR),
		list("Pill Bottle (Iron)", 40, /obj/item/storage/pill_bottle/iron, null, VENDOR_ITEM_REGULAR),
```

**Comment by DexterDude** [comment]
Maybe a nitpick, but I would prefer vendors not be in front of windows unless there's no other option.

**Comment by Zonespace27** [comment]
> Maybe a nitpick, but I would prefer vendors not be in front of windows unless there's no other option. Just looks a bit visually jarring. Maps are fine otherwise!

Moved em around a bit

**Review by morrowwolf — DISMISSED** [review]
Fine with me 👍

**Comment by private-tristan** [comment]
Should also remove the empty reagent pouch from the corpsman loadout vendor, as it's useless with this change.

**Comment by Zonespace27** [comment]
> Should also remove the empty reagent pouch from the corpsman loadout vendor, as it's useless with this change.

done

**Comment by DexterDude** [comment]
I think IA should be available by default, otherwise, I think it's fine to merge at this point.

**Pull request merged**